STEP/AST/Nodes/IdNode.cs

```diff
@@ -13,7 +13,8 @@ public override bool Equals(object obj)
         if (obj is IdNode other)
         {
             return Equals(other.Id, Id)
-                && Equals(other.AttributesRef, AttributesRef);
+                && Equals(other.AttributesRef, AttributesRef)
+                && Equals(other.Type, Type);
```

Comment:
Er stadig lidt usikker på hvordan dette other.Type skal bruges
![ijiji](https://user-images.githubusercontent.com/72670627/165245643-d191f744-df04-42e1-b05d-5261b8a5947e.jpg)

Reply:
Metodens signatur er Equals(object obj) - 
if (obj is IdNode other) er true hvis obj er en IdNode, og det laver i samme step et alias dertil "other".
Equals(other.Type, Type) sammenligner Type på det nuværende object og det object Equals metoden bliver kaldt med (other).